### PR TITLE
chore: log exception when cannot create data and add SDK logging configuration

### PR DESCRIPTION
**Pull request by shimoncohen**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: MergerService/Properties/launchSettings.json [hunk 1]
```diff
@@ -14,7 +14,11 @@
       "launchBrowser": true,
       "launchUrl": "swagger",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "AWS_ACCESS_KEY_ID": "minio123",
+        "AWS_SECRET_ACCESS_KEY": "minio123",
+        "S3__url": "http://localhost:9000",
```
Comment (asafMasa) [inline MergerService/Properties/launchSettings.json:20]: "AWS_REGION": "custom", ?

File: MergerService/appsettings.json [hunk 2]
```diff
@@ -17,7 +17,11 @@
   },
   "S3": {
     "url": "http://localhost:9000",
-    "bucket": "tiles"
+    "bucket": "tiles",
+    "request": {
+      "timeoutSec": 5,
+      "retries": 3
+    }
```
Comment (asafMasa) [inline MergerService/appsettings.json:24]: ? "logToConsole": false

**Review by asafMasa — CHANGES_REQUESTED** [review]
Well done

File: MergerService/Properties/launchSettings.json (hunk 1 above)
Comment (shimoncohen) [reply MergerService/Properties/launchSettings.json:20]: Doesn't matter... these are some launch settings for Rider

File: MergerService/appsettings.json (hunk 2 above)
Comment (asafMasa) [reply MergerService/appsettings.json:24]: to debug purposes

**Review by asafMasa — APPROVED** [review]

**Pull request merged**